docs/controllers/scaling.md

```diff
@@ -0,0 +1,73 @@
+# Scaling
+
+This chapter is about strategies for scaling controllers and the tradeoffs these strategies make.
+
+## Motivating Questions
+
+- Why is the reconciler lagging? Are there too many resources being reconciled?
+- What happens when your controller starts managing resource sets so large that it starts significantly impacting your cpu or memory use?
+- Do you give your more resources?
+- Do you add more pods? How can you do this safely?
```

Comment:
Should these be given an indent? They feel related to the previous point of cpu / mem saturation.